global:
  target_ip: 192.168.178.70 

streams:
  color_camera_info:
    type: zenoh
    topic: /camera/camera/color/camera_info
    msg_type: sensor_msgs/CameraInfo
    allowed_in_state: [ALL]
  network_information:
    type: zenoh
    topic: /network_state
    msg_type: sensor_msgs/CameraInfo
    allowed_in_state: [EXCELLENT]
  depth_camera_info:
    type: zenoh
    topic: /camera/camera/depth/camera_info
    msg_type: sensor_msgs/CameraInfo
    allowed_in_state: [ALL]
  usb_cam_video4:
    type: gstreamer
    device: /dev/video2
    port: 5003
    width: 640
    height: 480
    min_bitrate: 1000
    max_bitrate: 4000
    allowed_in_state: [ALL]
  usb_cam_video5:
    type: gstreamer
    device: /dev/video4
    port: 5004
    width: 640
    height: 480
    min_bitrate: 1000
    max_bitrate: 4000
    allowed_in_state: [ALL]
